deps.edn

```diff
@@ -1 +1,6 @@
-{:deps {io.github.manateelazycat.cloel/cloel {:mvn/version "1.0.0"}}}
+{:deps {io.github.manateelazycat.cloel/cloel {:mvn/version "1.0.0"}}
+
+ :paths ["src"]
+
+ :aliases
+ {:run-by-cloel {:main-opts ["-m" "reorderfile"]}}}
```

Comment:
`:main-opts ["-m" "reorderfile"]` means call the `-main` function in `reorderfile` namespace with command line args as function args